kong/utils.go

```diff
@@ -278,7 +278,7 @@ func backfillResultConfigMap(res Configuration, path []string, configValue inter
 	return nil
 }
 
-func fillConfigRecord(schema gjson.Result, config Configuration) Configuration {
+func fillConfigRecord(schema gjson.Result, config Configuration, kongConfig Configuration) Configuration {
```

Comment:
naming: The `kongConfig` seems to infer to the existing configuration. The `config` and `kongConfig` looks a bit confusing to me, could we name it `oldConfig` or `existingConfig` and add comments to explain them?

Reply:
done.